src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -314,6 +315,33 @@ public Job call() throws SaltStackException {
     }
 
     /**
+     * Get previously run jobs.
+     * @return map containing run jobs keyed by job id.
+     * @throws SaltStackException if anything goes wrong
+     */
+    public Map<String, Job> getJobs() throws SaltStackException {
+         Result<List<Map<String, Job>>> result = connectionFactory
+                .create("/jobs", JsonParser.JOBS, config)
+                .getResult();
+         return result.getResult().get(0);
```

Comment:
Not too sure if it **needs** to be more safe, it actually depends on what the server is returning. If a helper method should be added then please via a separate PR.